tests/snappi/conftest.py

```diff
@@ -76,3 +76,30 @@ def enable_packet_aging_after_test(duthosts, rand_one_dut_hostname):
 
     duthost = duthosts[rand_one_dut_hostname]
     enable_packet_aging(duthost)
+
+
+@pytest.fixture(scope="module")
+def retrieve_pfc_pause_quanta(duthosts, rand_one_dut_hostname):
```

Comment:
Where is this fixture getting called?

Reply:
This is a typo, I changed it in another branch but forgot to update this one, will update